src/components/chip-list/chip-list.js

```diff
@@ -27,11 +27,13 @@ const ChipList = ({
                     />
                     <div className="chip-holder-betting-chips">
                         {hasEnabledChips &&
```

Comment:
Borde vi inte flytta denna failsafe currentBettingChips?.chips || {}
```suggestion
                        {hasEnabledChips &&
```

Till rad 29 istället?

Onödigt o köra map på en tom array imo :)

Reply:
Sant! Fixar det